trainingMaquetado/maquetado.css

```diff
@@ -1,3 +1,23 @@
+.paragraph {
+  color: #999;
+  font-family: "Open Sans";
```

Comment:
`font-family: "Open Sans", sans-serif;`

add sans-serif as a default font in case open sans doesn't load.